src/reanimated2/initializers.ts

```diff
@@ -77,15 +77,15 @@ function valueUnpacker(objectToUnpack: any, category?: string): any {
       handleCache!.set(objectToUnpack, value);
     }
     return value;
-  } else if (category === 'RemoteFunction') {
+  } else if (category === 'RNFunctionRef') {
```

Comment:
I believe this is the only use case for `category`, do you think there's more use cases for this or can we convert it to some boolean flag?

Reply:
I think the idea behind it was to use in more places but eventually this is the only one. After I double check it (and I guess you too) I think we could move it to such flag.